### PR TITLE
Add tests to verify the origin of Blob URLs.

### PR DESCRIPTION
**Pull request by mkruisselbrink**
Tests for the change from unicode serialization to ascii serialization in https://github.com/w3c/FileAPI/pull/61

### CODE REVIEW

File: FileAPI/url/origin.sub.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      if (e.source != frame.contentWindow) return;
+      const url = e.data.url;
+      assert_false(url.includes('天気の良い日'),
+          'Origin should be ascii rather than unicode');
+      assert_equals(new URL(url).origin, e.origin,
+          'Origin of URL should match origin of frame');
+      assert_true(url.startsWith('blob:{{location[scheme]}}://xn--'));
+      t.done();
+    }));
+  frame.addEventListener('load', t.step_func(() => {
```
Comment (Ms2ger) [inline FileAPI/url/origin.sub.html]: I think adding a load event handler here is vulnerable to a race condition where the event would be dispatched before the script is executed.

File: FileAPI/url/origin.sub.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<iframe src="{{location[scheme]}}://{{domains[天気の良い日]}}:{{location[port]}}/FileAPI/support/url-origin.html"></iframe>
+<script>
+test(t => {
+  const blob = new Blob(["Test Blob"]);
+  const url = URL.createObjectURL(blob);
+  assert_equals(new URL(url).origin, location.origin);
+  assert_true(url.includes(location.origin));
```
Comment (Ms2ger) [inline FileAPI/url/origin.sub.html]: Also add a `startsWith` check here?

File: FileAPI/url/origin.sub.html (hunk 1 above)
Comment (zcorpan) [inline FileAPI/url/origin.sub.html]: Fix would be to create the iframe with `createElement` and append it to body.

File: FileAPI/url/origin.sub.html (hunk 2 above)
Comment (mkruisselbrink) [inline FileAPI/url/origin.sub.html]: Done

File: FileAPI/url/origin.sub.html (hunk 1 above)
Comment (mkruisselbrink) [inline FileAPI/url/origin.sub.html]: Changed it to be that way. I'm always confused if it is possible or not for the load event to be fired before the first synchronous javascript executes. But dynamically creating the iframe is definitely cleaner.

**Pull request merged**